docs/docs/api/01-API/table-of-contents.md

```diff
@@ -230,3 +230,9 @@ displayed_sidebar: apiSidebar
 
 -  [notExists](../checks/notexists)
 
+## Getters
+
+-  [getAll](../getters/getall)
+
+-  [get](../getters/get)
```

Comment:
@mlikasam-askui 

I thought you've changed the naming of the folders?

Reply:
it was not needed